### DISCUSSION TITLE
Multi github account

### DISCUSSION THREAD

**Discussion by janabdullahi** [dailydotdev/daily#900] [Feature Requests]
I want to add my daily dev account in a multiple github account.
Is there any way to add it in my different github accounts ?

**Comment by idoshamun** [comment]
Hi @janabdullahi,

At this point, we support only one account per provider.
Can you elaborate on your use case and provide more context?

**Reply by janabdullahi** [reply]
I have two github account one is offical and another is my personal so I want to use it in my both accounts.
